elyra/pipeline/resources/palette.json

```diff
@@ -104,7 +104,7 @@
       ]
     },
     {
-      "label": "R Script",
+      "label": "R",
       "image": "",
       "id": "r-script",
       "description": "R Script",
```

Comment:
If this is the component description in the palette, it should probably read "Run R script" because a node implements a task, not a file of that type.

Reply:
@kiersten-stokes why we change these? Could we leave it the way it is on master?

Reply:
I think the change was to force the names to match how the properties for the generic components were originally rendered. Specifically, the filename parameter, which has the below description:

`The path to the [filetype] file.`

I guess the best solution would be to just remove the word `file` from that description and we can change the labels back to how they were originally.

Reply:
Same answer as above